src/components/badge/badge.scss

```diff
@@ -0,0 +1,25 @@
+@import '../../style/variables';
+
+/**
+ * @prop --badge-background-color: badge background color
+ * @prop --badge-text-color: badge text color
+ */
+
+
+:host {
+  display: block;
+}
+
+.badge-container {
+    display: inline-block;
+    font-size: 0.7rem;
+    min-width: 1.2rem;
+    height: 1.2rem;
+    border-radius: 1.2rem;
+    line-height: 1.2rem;
+    text-align: center;
+    background-color:  var(--badge-background-color, #EEF1F5);
+    color:  var(--badge-text-color, #7D899C);
+    padding: 0 0.3rem;
+    margin-left: 0.25rem;
```

Comment:
Is there any specific reason why none of the size values use the `pxToRem` function, and why almost all of them are set at values that result in fractions of pixels at the default size of 16px/rem? @Befkadu1

Reply:
Tweaking this in #678